osbs/cli/main.py

```diff
@@ -704,16 +704,16 @@ def cli():
                               help="perform a scratch build")
     build_parser.add_argument("--yum-proxy", action='store', required=False,
                               help="set yum proxy to repos from koji/add-yum-repo params")
+    build_parser.add_argument('--release', action='store', required=False,
+                              help='release value to use')
+    build_parser.add_argument('--arrangement-version', action='store', required=False,
+                              help='version of inner template to use')
```

Comment:
Huh.

Well, I don't really mind. Given that it's not changing what we had anyway, maybe this PR is fine as-is, moving `--arrangement-version` out of the group parser.